db/misc/20201210144200.create-analysis-users.sql

```diff
@@ -0,0 +1,22 @@
+/* example script for creating analysis users and assigning to roles */
```

Comment:
If the intention is that this is a script that will not be run automatically, I think the intention would be clearer if we moved this to markdown and put it in the docs/ directory.

Reply:
none of the scripts in this directory are automatic.  they are manual utility scripts that we have collected over time.

docs area doesn't feel like the right place for these.  they should be available as sql.

i have included a readme in this folder to indicate that they are manual

Reply:
Well you certainly got my heart rate up when I saw empty passwords until I realised this was in a different directory 😆